app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,89 @@
+class ReviewsController < ApplicationController
+
+  def index
+    @reviews = Review.all
+    @reviews_page = @reviews.page(params[:page])
+    @reviews = Review.search(params[:search])
+    respond_to do |format|
+      format.html
+      format.csv
+      format.xls  #{ send_data @universities.to_csv(col_sep: "\t") }
+    end
+  end
+
+  def show
+    @review = Review.find(params[:id])
+  end
+
+  def new
+    @review = Review.new
+    @university = University.all
+  end
+
+  def edit
+    @review = Review.find(params[:id])
+    count_total
+  end
+
+  def create
+    @review = Review.new(resource_params)
+    count_total
+    if @review.save
+      redirect_to @review
+    else
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.